CHANGELOG.md

```diff
@@ -32,7 +37,6 @@
 ### Bugfix
 
 - Fix loading of cookie on SSR for certain requests, revert slight change in how they are loaded introduced in alpha 16 @sneridagh
-- Fix disable mode of `QuerystringWidget` when all criteria are deleted @kreafox
```

Comment:
@nzambello can you elaborate on why you moved this changelog entry? This already has been released if I am not mistaken.

Reply:
No idea, I guess a merge conflict wrong resolution. Fixed it